src/main/java/io/apiman/cli/command/api/command/ApiCreateCommand.java

```diff
@@ -62,6 +67,9 @@
     @Option(name = "--gateway", aliases = {"-g"}, usage = "Gateway")
```

Comment:
ok I remove the "gateways" arg and make the "gateway" one, being a List (can occurs more than once)
for instance : "-g gw1 -g gw2"